plugins/filetrees/neo-tree.nix

```diff
@@ -57,6 +57,16 @@ in
 
       package = helpers.mkPluginPackageOption "neo-tree" pkgs.vimPlugins.neo-tree-nvim;
 
+      gitPackage = helpers.mkPackageOption {
+        name = "git";
+        default = pkgs.git;
+      };
```

Comment:
This is from #2113 right ?

Reply:
I can move it into that one, but I didn't touch that plugin in that PR.

Reply:
I'd prefer to only have iconPackages in this PR, it's better for the git history

Reply:
Alright, I'll suffle the changes around a bit.